tests/entrypoints/openai/test_transcription_validation.py

```diff
@@ -23,7 +23,10 @@
 
 
 @pytest.fixture(scope="module")
-def server():
+def server(request):
+    if marker := request.node.get_closest_marker("extra_server_args"):
+        SERVER_ARGS.append(marker.args[0])
+
```

Comment:
sorry about the delay, I hadn't checked tests thoroughly, I don't think this is actually working as intended.
As the fixture is per-module, it is going to be created once and then re-used for all other tests. 
What's happening here is that it's using the first tests args to spawn the server, so it's dependent on the order of tests.

Reply:
@max-wittig let's try and spawn the module and particularly the newly added function fixture with minimal `--gpu-memory-utilization` and args.